Source/WebKit/WebProcess/WebCoreSupport/WebPermissionController.cpp

```diff
@@ -121,21 +122,24 @@ void WebPermissionController::tryProcessingRequests()
     }
 }
 
-void WebPermissionController::addObserver(WebCore::PermissionObserver& observer)
+void WebPermissionController::addObserver(WebCore::PermissionStatusIdentifier statusIdentifier, WebCore::ClientOrigin& origin, WebCore::PermissionDescriptor& descriptor, WebCore::ScriptExecutionContextIdentifier contextIdentifier)
```

Comment:
Let's use a const reference for `origin` instead of a non-const one.

No need to pass `descriptor` by reference, it is just an enum type.

Reply:
Fixed!